Modules/Scripted/Home/Home.py

```diff
@@ -1036,17 +1036,23 @@ def onResetContrastButtonClicked(self):
     self.get('ContrastSlider').setValues(self.logic.DefaultWindowLevelMin, self.logic.DefaultWindowLevelMax)
 
   def onThicknessChanged(self, value):
-    self.Annotations.current.thickness = value
-    self.Annotations.current.update()
+    if self.Annotations.current:
+      self.Annotations.current.thickness = value
+      self.Annotations.current.update()
+
+    ClosedCurveAnnotation.DefaultThickness = value
```

Comment:
Should we apply the same approach to `DefaultOntology`, `DefaultReferenceView` and `DefaultStepSize` ?

Reply:
Done in f213032.